internal/controller/http/restrouter.go

```diff
@@ -53,7 +53,7 @@ func (c *RestController) SetCustomConfigInfo(customConfig interfaces.UpdatableCo
 }
 
 func (c *RestController) InitRestRoutes() {
-	c.lc.Info("Registering v2 routes...")
+	c.lc.Info("Registering v3 routes...")
```

Comment:
```suggestion
	c.lc.Info("Registering routes...")
```

Let's remove the version number here to avoid the future missing
Thanks for catching this up

Reply:
The commit message should also be modified to:
"refactor: Remove the version number in Init logs"

Reply:
Thanks for your feedback. I've made the suggested changes and updated the commit message.